pkg/kubelet/kubelet.go

```diff
@@ -796,11 +809,13 @@ func (kl *Kubelet) validateNodeIP() error {
 	return fmt.Errorf("Node IP: %q not found in the host's network interfaces", kl.nodeIP.String())
 }
 
+// allSourcesReady returns whether all seen pod sources are ready.
```

Comment:
in case someone looks at this PR, I have a PR to just move sources ready
https://github.com/kubernetes/kubernetes/pull/24810